src/mrinufft/trajectories/utils.py

```diff
@@ -41,6 +59,81 @@ class Gammas(float, Enum):
     X = X129 = XENON
 
 
+class Spirals(FloatEnum):
+    """Enumerate spirals type."""
+
+    ARCHIMEDES = 1
+    ARITHMETIC = ARCHIMEDES
+    FERMAT = 2
+    PARABOLIC = FERMAT
+    HYPERBOLIC = -1
+    LITHUUS = -2
+
+
+class NormShapes(FloatEnum):
+    """Enumerate norms types."""
```

Comment:
"Enumerate shape norms."
Also I wonder if we should just name it "Shapes" to keep it similar to "Spirals", since both actually relate to values in equations